docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -157,15 +157,13 @@ dune --deploy ./build/hello hello
 
 > 👀 **Code Permission**
 > 
-> By default, DUNE adds the `eosio.code` permission to an account when you deploy a contract to it. This allows the
-> contract to be able to trigger inline actions on other smart contracts.
+> By default, DUNE adds the `eosio.code` permission to an account when you deploy a contract to it. This allows the contract to trigger inline actions on other smart contracts.
 
 ### Interacting with your contract
 
-To interact with your contract you will send a transaction on your local EOS node. Transactions on EOS are made up of 
-`actions`, so we will send a single action to your contract.
+To interact with your contract send a transaction on your local EOS node. Transactions on EOS are made of `actions`. Send a single action to your contract.
 
-We will also create a test account to send the action from.
+You can also create a test account from which to send the action.
```

Comment:
Can should be will/must. 

It's not optional here, otherwise the rest will fail

Reply:
Done